ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/ConfigEnricher.scala

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.benchtool
+
+import com.daml.ledger.api.benchtool.config.WorkflowConfig.StreamConfig
+import com.daml.ledger.api.benchtool.submission.CommandSubmitter
+import com.daml.ledger.api.v1.value.Identifier
+import com.daml.ledger.test.model.Foo.{Foo1, Foo2, Foo3}
+import scalaz.syntax.tag._
+
+object ConfigEnricher {
+
+  def enrichedStreamConfig(
```

Comment:
This method enriches short party and template names with appropriate suffixes that are generated in the submission phase. For example, one can defined a submission phase using `Obs-1` as party. In reality a party name could be something like `Obs-1-1c33fa656d54e` with a generated suffix for uniqueness. `ConfigEnricher` will take care to pass correct party name.

Reply:
Please let me know if should put such comment in the code as well.

Reply:
That's fine as is actually

Reply:
I would use imperative throughout the method names though
```suggestion
  def enrichStreamConfig(
```

Reply:
Fixed 👍